assets/player/skins/fishy/fishy.player.yaml

```diff
@@ -234,9 +234,6 @@ layers:
       emote_alarm:
         fps: *fps
         frames:
-          - 0
-          - 0
-          - 0
```

Comment:
we should remove these for the other players too.

Reply:
Removed!